runbooks/OutdatedVirtualMachineInstanceWorkloads.md

```diff
@@ -0,0 +1,61 @@
+# OutdatedVirtualMachineInstanceWorkloads
+
+## Meaning
+
+There are VMIs running in outdated virt-launcher pods 24 hours after the KubeVirt control plane update has completed.
+
+## Impact
+
+VMIs that have not been updated to run in the most recent virt-launcher pod may not have access to new KubeVirt features and will not have received any security fixes associated with the virt-launcher pod update.
+
+## Diagnosis
+
+You can identify the VMIs that are out-of-date by using the `kubevirt.io/outdatedLauncherImage` label as a label selector when listing VMIs. Below is an example of a command that will list all out-of-date VMIs across all namespaces within the cluster.
+
+```bash
+kubectl get vmi -l “kubevirt.io/outdatedLauncherImage” --all-namespaces
+```
+
+## Mitigation
+
+### Check if automated workload updates are enabled
+
+Check the KubeVirt CR used to install KubeVirt to see if the `workloadUpdateStrategy` is configured on the KubeVirt CR's spec.
```

Comment:
What about adding a link to https://kubevirt.io/user-guide/operations/updating_and_deletion/#updating-kubevirt-workloads ?

Reply:
fixed